cddl/realm/cca-realm-claims.cddl

```diff
@@ -9,5 +9,5 @@ cca-realm-claim-map = {
     cca-realm-hash-algo-id
     cca-realm-public-key
     cca-realm-public-key-hash-algo-id
-    cca-realm-mec-policy
+    ? cca-realm-mec-policy
```

Comment:
NOTE: I had to relax this because the TF-M example does not include the MEC policy claim. Maybe we could explicitly mark this as "temporary workaround".

Reply:
Yes - this is new in 1.1 (where it is mandatory). Let's mark this as a workaround until full upgrade to 1.1 can be done (later)

Reply:
you're right, that's probably cleaner

Reply:
done in [8c2acf7](https://github.com/SimonFrost-Arm/draft-ffm-rats-cca-token/pull/25/commits/8c2acf791a4f5348c7361c9137eb93099e8ebbe9)